docs/user-guide/sns-instruments/vanadium_processing.ipynb

```diff
@@ -0,0 +1,395 @@
+    "\n",
+    "First, define estimates for the peaks based on the known crystal structure of vanadium:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7140935e-8b32-403f-a0d8-e66293819b40",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "peak_estimates = powgen.peaks.theoretical_vanadium_dspacing(\n",
+    "    hkl_range=7, min_d=sc.scalar(0.41, unit='angstrom')\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e530963-258e-475f-9128-2417bc8d7e24",
+   "metadata": {},
+   "source": [
+    "We need to histogram the data to perform fits:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d6bcd10f-130e-43e0-8231-1f0b70e8bc53",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "peak_histogram = peaked_data.hist()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1c4ff398-80d4-4452-92d7-3d3c8d1359dc",
+   "metadata": {},
+   "source": [
+    "The fits require a bin-center coordinate, so convert from bin-edges:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e661fda9-1de4-4d26-a8d7-c84cec10decd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "to_fit = peak_histogram.copy(deep=False)\n",
+    "to_fit.coords['dspacing'] = sc.midpoints(to_fit.coords['dspacing'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b0bbf51f-307f-43c5-8093-ff6606049a1f",
+   "metadata": {},
+   "source": [
+    "Perform the fits:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "49f574d8-de6f-4c69-9704-6ce4acb9bdb3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fit_results = powgen.peaks.fit_vanadium_peaks(to_fit, peak_estimates=peak_estimates)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "19251c69-75eb-4dd0-be97-d2272946bacd",
+   "metadata": {},
+   "source": [
+    "Remove the fitted peaks to obtain the incoherent scattering.\n",
+    "Also restore the bin-edge coordinate that we had to replace temporarily for the fits.\n",
+    "\n",
+    "Importantly, we remove variances from the data.\n",
+    "If we kept the variances, subtracting the fitted models would introduce correlations between the data points.\n",
+    "This corresponds to [UncertaintyBroadcastMode.drop](../../generated/modules/ess.powder.types.UncertaintyBroadcastMode.rst) in the main workflow.\n",
+    "See also the [guide in ESSreduce](https://scipp.github.io/essreduce/user-guide/reduction-workflow-guidelines.html#s-8-propagation-of-uncertainties-in-broadcast-operations-should-support-drop-and-upper-bound-strategies-upper-bound-shall-be-the-default)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fbcde35f-8608-4ecb-9b6d-67f686d7839a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "incoherent = scn.peaks.remove_peaks(sc.values(to_fit), fit_results)\n",
```

Comment:
[Unrelated to this PR] Are the `fit_vanadium_peaks` and `scn.peaks.remove_peaks` designed to be used in pairs, on the same data, or could you imagine have some `fit_results` from another run and you use them to `remove_peaks` on a new Vanadium data?
I guess the height of the peaks etc would become an issue?

Reply:
Yes and it is also useful for making the diagnostics plot.

Should we add those functions into a workflow graph so we have a workflow which computes a "vanadium without peaks" or something like that?

Reply:
In the reflectometry workflow, johannes added some types which were plots instead of data. You could then `pipeline.compute(DiagnosticPlot)` to view the figure.
Maybe something like this coudl also work here?

That said, I'm not so bothered, leaving it as is also works.

Reply:
This could be done for diagnostics. But we may still need to modify the fit results before removing peaks. I'd say we leave it as is until we have more insight. In particular from DREAM.